stronk/schemas/workout/mutations.py

```diff
@@ -61,11 +67,13 @@ def mutate(root, info, id: str, name: str = None, description: str = None, proje
             attrs['description'] = description
         if projected_time:
             attrs['projected_time'] = projected_time
+        if scheduled_time:
+            attrs['scheduled_time'] = date_time_str_to_date(scheduled_time)
```

Comment:
Should we cast `scheduled_time` to a datetime object just like we did for CreateWorkout for consistency? Also I don't think the `update` method in the workout model.